tank/model_utils.py

```diff
@@ -57,8 +59,81 @@ def get_hf_model(name):
     # TODO: Currently the test input is set to (1,128)
     test_input = torch.randint(2, (1, 128))
     actual_out = model(test_input)
+
+    # fx_g = make_fx(
+    #     model(test_input),
```

Comment:
You just need to pass the model here.

Reply:
Can we resolve and get checks green for now?

Reply:
Yes

Reply:
> Can we resolve and get checks green for now?

yes